index.js

```diff
@@ -88,6 +88,8 @@ module.exports = function () {
     if (!system.global.hasOwnProperty('monitor')) { system.global.monitor = true }
     if (!system.global.hasOwnProperty('monitor_excludes')) { system.global.monitor_excludes = [] }
     if (!system.global.hasOwnProperty('dns_enabled')) { system.global.dns_enabled = false }
```

Comment:
yeah I saw that was why - another way is if `(!('prop' in system.global'))`
I'll take a look anyway